### DISCUSSION TITLE
How to give a line a color in a list view?

### DISCUSSION THREAD

**Discussion by LaurensCleyman** [YetiForceCompany/YetiForceCRM#15799] [How to]
I saw following screenshot online:

![image](https://user-images.githubusercontent.com/16014524/159288613-6d2edb57-855a-4b19-8b53-191d361e83c0.png)

How can we give a whole line a certain color?

**Comment by mariuszkrzaczkowski** [comment]
such a mechanism was available in older versions now no longer exists

**Comment by vovpff** [comment]
This can be done by programmer.
